### DISCUSSION TITLE
API Token and LDAP Groups?

### DISCUSSION THREAD

**Discussion by jemery27** [netbox-community/netbox#6395] [Q&A] [answered]
Hi - we use netbox with LDAP auth and LDAP groups configured in netbox with permissions assigned to the groups. I think this is working fairly well although the netbox web interface doesn't show group membership but we manage that in AD. But now we have a couple users trying to hit the API and their tokens only return that they don't have permissions to view any data. If I add them to their matching LDAP group within netbox manually, then their tokens work. Am I missing some configuration or something with LDAP groups that would allow the API to use the LDAP group membership or does that have to be managed manually?

Thanks,
Jason

**Comment by jemery27** [comment]
Can anyone confirm or deny LDAP group permissions work or don't work with the netbox API? Just trying to figure out if this is a configuration issue or just functional limitation.

Thanks,
Jason

**Comment by tyler-8** [comment, marked as answer]
If you use group mirroring, it will work. See https://github.com/netbox-community/netbox/issues/5442#issuecomment-770997825

Related issue: https://github.com/netbox-community/netbox/issues/3063

There is a PR that _might_ solve this issue but I haven't tested that myself just yet 
https://github.com/netbox-community/netbox/pull/6580
